src/index.html

```diff
@@ -6,9 +6,24 @@
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Moyo header</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500&display=swap&subset=cyrillic" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Moyo header</h1>
+  <header class="header">
+    <a class="logo" href="/"><img src="images/logo.png" alt="logo" width="40px" height="40px"></a>
+    <nav class="menu">
+      <ul class="menu-list">
+        <li class="menu-item"><a class="menu-link is-active" href="/">Apple</a></li>
```

Comment:
One tag - one line. Use new lines for nested tags. Don't write nested tags in one line.

Reply:
Ok, I will remember. Thanks